ops/ansible/playbooks-ccs/build_bfd-server.yml

```diff
@@ -18,6 +18,15 @@
       include_vars:
         dir: vars/{{ env }}
       tags: [pre-ami, post-ami]
+
+    # Before the bfd-server role copy the keystore file 
+    - name: Copy the appserver keystore  
+      copy:
+        # Note: this JKS file is encrypted via the Ansible Vault 
+        src: "{{ data_server_appserver_keystore }}"
+        dest: '/jboss/bluebutton-appserver-keystore.jks'
+      tags:
+        - pre-ami
```

Comment:
Doesn't this need to happen during instance launch rather than AMI build to get the keystore for the correct environment?

Reply:
According to @jzulim, we are building a different AMI for each environment. See the `Copy Local Test SSL Keypair` task for the pattern that I'm following.

Reply:
I thought the plan was to move away from that, but it's fine if that's out of scope for this PR

Reply:
@jeskew-gov you're absolutely right we need to move away from this, but as you mentioned, slightly out of scope considering our current deployment strategy. But since we are building unique AMI's and will probably be doing so for a while (dust needs to settle) following the current model seems like the right thing to do for the moment.